src/containers/shared/components/Transaction/OracleDelete/Simple.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const { OracleDocumentID } = data.instructions
+  return (
+    <div className="oracle-delete">
+      {OracleDocumentID !== undefined && (
```

Comment:
Isn't `OracleDocumentID` a required field in the transaction?

Reply:
It could be `0`